addons/point_of_sale/models/pos_config.py

```diff
@@ -912,13 +912,13 @@ def load_onboarding_clothes_scenario(self, with_demo_data=True):
             'record': config,
             'noupdate': True,
         }])
-        if with_demo_data:
-            config._load_onboarding_clothes_demo_data()
+        config._load_onboarding_clothes_demo_data(with_demo_data)
         return {'config_id': config.id}
 
-    def _load_onboarding_clothes_demo_data(self):
+    def _load_onboarding_clothes_demo_data(self, load_products=True):
```

Comment:
```suggestion
    def _load_onboarding_clothes_demo_data(self, with_demo_data=True):
```

Rename the argument because the xml file you are loading contains types of records other than products.